package.json

```diff
@@ -73,7 +73,7 @@
     "react-icon-base": "^2.0.4",
     "react-lazy-load": "3.0.10",
     "react-mentions": "^1.0.0",
-    "react-router": "2.8.1",
+    "react-router": "3.0.2",
```

Comment:
Can we use the latest version that includes react 16 support (3.2.0)?

Reply:
Yes!